src/rodal.js

```diff
@@ -79,6 +87,7 @@ class Rodal extends React.Component {
     measure: 'px',
     visible: false,
     showMask: true,
+    close: null,
```

Comment:
How about rename to `closeIcon`? same as [ant-design](https://ant.design/components/modal)

Reply:
Sounds good. It's much clear than just naming close